### DISCUSSION TITLE
about Kyber768Draft00

### DISCUSSION THREAD

**Discussion by swoky** [bcgit/bc-java#1920] [General]
i see the release notes in 1.72

`The CRYSTALS-Kyber Finalist NIST Post Quantum Algorithm has been added to the low-level API and the BCPQC provider.`

dose the kyber version  in release 1.72 match Kyber768Draft00 ?

**Comment by dghgit** [comment]
Please note the NIST IP release statement does not cover Kyber now that FIPS PUB 203 has been released, only the ML-KEM algorithm is covered by the IP release statement. For avoidance of confusion, yes IP stands for Intellectual Property, I would guess you will be able to work out the rest for yourself.

I would strongly recommend using ML-KEM, and telling whoever you are trying to be compatible with to do likewise.
